package.json

```diff
@@ -29,6 +29,7 @@
     "heroku-client": "3.0.2",
     "mocha": "3.2.0",
     "nock": "9.0.11",
+    "np": "^2.15.0",
```

Comment:
typically we save exact versions, so yeah, it's a good idea. In practice though, you could either edit the yarn.lock file OR make this exact and it would achieve the same result (though not for transitive dependencies)